### PR TITLE
Feature/transforms

### PR DESCRIPTION
**Pull request by pcholuj**
Add new Filelink class for better transform handling

### CODE REVIEW

File: src/lib/filelink.ts [hunk 1]
```diff
@@ -0,0 +1,1332 @@
...
+  /**
+   * Adds polaroid transformation
+   *
+   * @see https://www.filestack.com/docs/api/processing/#polaroid
+   * @param {(PolaroidParams | boolean)} params
+   * @returns this
+   * @memberof Filelink
+   */
+  polaroid(params?: PolaroidParams | boolean) {
+    return this.addTask('polarid', params);
```
Comment (gr00by87) [inline src/lib/filelink.ts]: polaroid

File: src/lib/filelink.ts [hunk 2]
```diff
@@ -0,0 +1,1332 @@
...
+  /**
+   * Adds oilpaint transformation
+   *
+   * @see https://www.filestack.com/docs/api/processing/#oilpaint
+   * @param {(OilPaintParams | boolean)} params
+   * @returns this
+   * @memberof Filelink
+   */
+  oilpaint(params?: OilpaintParams | boolean) {
+    return this.addTask('oilpaint', params);
```
Comment (gr00by87) [inline src/lib/filelink.ts]: oil_paint

File: src/lib/filelink.ts [hunk 3]
```diff
@@ -0,0 +1,1332 @@
...
+   * @memberof Filelink
+   */
+  pixelate(params?: PixelateParams | boolean) {
+    return this.addTask('pixelate', params);
+  }
+
+  /**
+   * Adds oilpaint transformation
+   *
+   * @see https://www.filestack.com/docs/api/processing/#oilpaint
```
Comment (gr00by87) [inline src/lib/filelink.ts]: #oil-paint

File: src/lib/filelink.ts [hunk 4]
```diff
@@ -0,0 +1,1332 @@
...
+   * @private
+   * @type {string}
+   * @memberof Filelink
+   */
+  private customDomain: string;
+
+  /**
+   * Class for generating tranformation urls
+   * @param {(string | string[])} source - handle or multiple handles (ie for collage)
+   * @param {string} [apikey] - your apikey - required fro all external sources
```
Comment (jar3k) [inline src/lib/filelink.ts]: fro -> from :)

File: src/lib/filelink.ts [hunk 5]
```diff
@@ -0,0 +1,1332 @@
...
+   *
+   * @private
+   * @type {string}
+   * @memberof Filelink
+   */
+  private customDomain: string;
+
+  /**
+   * Class for generating tranformation urls
+   * @param {(string | string[])} source - handle or multiple handles (ie for collage)
```
Comment (jar3k) [inline src/lib/filelink.ts]: ie -> i.e. :)

**Pull request merged**